example/lib/screens/message_view.dart

```diff
@@ -59,9 +82,34 @@ class _MessageViewState extends State<MessageView>  {
     });
   }
 
+  onLoadStarted() {
+    setState(() {
+      isLoading = true;
+    });
+  }
+
   onLoadFinished() {
     setState(() {
       isLoading = false;
     });
   }
+
+  onLoadError(PlatformException e) {
+    setState(() {
+      isLoading = false;
+      showDialog(
+          context: context,
+          builder: (context) => AlertDialog(
+            title: Text(e.message != null ? e.message : ""),
```

Comment:
Should you default the message to something? Otherwise, the user could see a dialog with no title and no content.